outputs.tf

```
@@ -0,0 +1,14 @@
+output "apigateway_vpc_link" {
+  description = "The VPC link to be used in REST API Gateways"
+  value       = aws_api_gateway_vpc_link.this
+}
+
+output "subnets" {
+  description = "Subnets used by the NLB"
+  value       = values(module.subnets.this)
```

@jansiwy This works:

```suggestion
  value       = [ for subnet in module.subnets : subnet.this ]
```

---

This should be the same:

```suggestion
  value       = values(module.subnets)[*].this
```